db/articles.js

```diff
@@ -0,0 +1,76 @@
+let articles = [];
```

Comment:
It's possible to actually make this a `const` where someone couldn't overwrite the variable with another type, like a String or Number. As a `const` you can still _add_ elements to an array though. Same with objects.